module/epochs/qc_client.go

```diff
@@ -26,13 +26,13 @@ type QCContractClient struct {
 	client            *client.Client // flow-go-sdk client to access node
 
 	// node details
-	accountAddress  string
+	accountAddress  sdk.Address
 	accountKeyIndex int
-	privateKey      crypto.PrivateKey
+	privateKey      string
 }
 
 // NewQCContractClient returns a new client to the QC contract
-func NewQCContractClient(accessAddress, qcContractAddress, accountAddress string, accountKeyIndex int, privateKey crypto.PrivateKey) (*QCContractClient, error) {
+func NewQCContractClient(privateKey, accountAddress string, accountKeyIndex int, accessAddress, qcContractAddress string) (*QCContractClient, error) {
```

Comment:
I would keep at least the private key as the more specific `PrivateKey` type and require the component setup code in `main.go` to handle decoding string inputs from the flags. Otherwise this component must be aware of the encoding scheme used by the CLI flags.